qiita_db/test/test_util.py

```diff
@@ -1172,13 +1172,7 @@ def test_BytesIO(self):
         with qdb.util.open_file(f) as fh:
             self.assertTrue(fh is f)
 
-    def test_hdf5IO(self):
```

Comment:
Not sure why we need this test but it seems like a duplicate of the one below ...

Reply:
Looks like this test is different from 1175, since here the test is checking for the identity of the filehandle, whereas the other test checks that the returned object is an instance of `File`. Maybe worth adding the assertion in 1178 to the test below.